config.js

```diff
@@ -676,6 +676,21 @@ var config = {
     // 'microphone', 'camera', 'select-background', 'invite', 'settings'
     // hiddenPremeetingButtons: [],
 
+    // Customize the Live Streaming dialog. Can be modified for a non-Youtube provider.
+    liveStreaming: {
```

Comment:
Please make the whole object here optional, aka, leave it commented out. It makes canging defaults much easier in the long run.